ssr-playground/rollup-server.config.js

```diff
@@ -0,0 +1,62 @@
+// This config is used when running `server.js`.
+
+import lwc from '@lwc/rollup-plugin';
+import replace from '@rollup/plugin-replace';
+import alias from '@rollup/plugin-alias';
+import simpleRollupConfig from'./rollup.config.js';
+
+const ENV = process.env.NODE_ENV ?? 'development';
+
+
+export default [
+    // Client-only build.
+    simpleRollupConfig({ watch: false }),
+
+    // Client build to rehydrate after SSR.
+    {
+        input: 'src/entry-client-ssr.js',
+        output: {
+          file: 'dist/entry-rehydrate.js',
+          format: 'cjs'
```

Comment:
This is written as `cjs` but consumed in a `<script>`. Can we use `esm` and `<script type="module">`?